demisto_sdk/__main__.py

```diff
@@ -138,9 +138,7 @@ def check_configuration_file(command, args):
 def main(config, version, release_notes):
     config.configuration = Configuration()
     import dotenv
-
-    from demisto_sdk.commands.common.tools import get_content_path
-    dotenv.load_dotenv(Path(get_content_path()) / '.env', override=True)  # Load a .env file from the cwd.
+    dotenv.load_dotenv(Path.cwd() / '.env', override=True)  # Load a .env file from the cwd.
```

Comment:
doesn't `dotenv` find the `.env` file automaticall?

Reply:
Perhaps `get_content_path() or Path.cwd()`?

Reply:
I think it should be the opposite:
`Path.cwd() or get_content_path()`
If `.env` is not found in the current path, search for `.env` in the content path.

Reply:
I meant that if `get_content_path()` returns `None`, it should be easy.
Otherwise, it should require a bit more code